### DISCUSSION TITLE
How can I get easyeffects to work with NixOS without home-manager?

### DISCUSSION THREAD

**Discussion by aboode95** [wwmm/easyeffects#2899] [Q&A] [answered]
I haven't setup home-manager in my system, so can someone guide me, how can I get easyeffects without home-manager?

**Comment by vchernin** [comment, marked as answer]
Do the official instructions work for you? https://search.nixos.org/packages?channel=23.11&show=easyeffects&from=0&size=50&sort=relevance&type=packages&query=easyeffects

**Reply by aboode95** [reply]
My apologies, I did try that and apparently the nixos-rebuild is breaking because of nvidia graphic driver update that requires a patch. I will accept yours as the correct answer (because it really is :3). I will only have to wait for the patch to come through.
